quicklogic/common/toolchain_wrappers/ql_symbiflow

```diff
@@ -44,6 +44,8 @@ OUT=()
 ROUTE_FLAG0=""
 MAX_CRITICALITY="0.0"
 JSON=""
+PNR_CORNER="slow"
+ANALYSIS_CORNER="fast"
```

Comment:
I'd set it to "slow" by default so that both PNR and analysis corners are the same.

Reply:
Done